tests/queries/0_stateless/02900_date_time_check_overflow.sql

```diff
@@ -0,0 +1,50 @@
+-- I hope we will get rid of "ignore" option one day :)
+-- It gives unpredictable result, but it is faster than others.
+SELECT 'ignore';
+SET date_time_overflow_mode = 'ignore';
+SELECT toDateTime(toDateTime64('1900-01-01 00:00:00.123', 3, 'UTC'), 'UTC');
```

Comment:
Could you please replace the (slightly unaesthetic) `UTC` timezone arguments in the function calls by a single `SET` statement at the beginning of the test which sets the session time zone? That will provide equal protection against time zone randomization.

Side note: https://clickhouse.com/docs/en/operations/settings/settings#session_timezone lacks an example for how to do just that (`SET` statement). Since doing so is a common use case, perhaps you would like to correct the documentation as well.